host-interaction/registry/offline-registry-manipulation.yml

```diff
@@ -0,0 +1,39 @@
+rule:
+  meta:
+    name: offline registry manipulation
+    namespace: host-interaction/registry
+    authOR: johnk3r
+    scope: function
+    att&ck:
+      - Discovery::Query Registry [T1012]
+    mbc:
+      - Operating System::Registry::Query Registry Value [C0036.006]
+      - Operating System::Registry::Create Registry Key [C0036.004]
+      - Operating System::Registry::Open Registry Key [C0036.003]
```

Comment:
I think we should split the apis and respective techniques create, query, open, etc. up into separate rules. Would you mind doing that? It's fine if we don't have samples for all and add those to the nursery.

Reply:
Good idea. Can I use the same sample if possible?

Reply:
Hello. Please evaluate the submitted rules. 

How can I do when a sample of a rule takes too long to process? As the pipeline does not accept samples that take a long time to process, should I create this rule as a nursery?

Reply:
yeah, putting the rule in the nursery is a fine choice.